src/app.config.ts

```diff
@@ -0,0 +1,108 @@
+import { isObject, isString } from './shared/types/types.helper';
+import { ENV } from './shared/env/env.contants';
```

Comment:
perhaps, it'd be beneficial to have a config service instead of just constant

Reply:
![image](https://user-images.githubusercontent.com/110594510/188175313-6cdd3c86-eff9-46ae-a5e9-0e268359e37d.png)

I encapsulated `dotenv.config()` call to `app.config.ts`, the main idea to use `AppConfig` like `const databaseConfig = new AppConfig().build();` to get env variables (and not only env variables) in every place of the project. It's supposed`AppConfig` only has access to env variables and other configurable data.